src/main/java/mate/academy/spring/model/ShoppingCart.java

```diff
@@ -15,7 +15,7 @@
     @Id
     private Long id;
     @OneToMany
-    private List<Ticket> tickets;
+    private List<Ticket> ticketsId;
```

Comment:
```suggestion
    private List<Ticket> tickets;
```

why ticketsId? You have just list of tickets